source/PhysicalQuantity/Time.php

```diff
@@ -71,6 +71,26 @@ protected static function initialize()
         $newUnit->addAlias('gregorian years');
         static::addUnit($newUnit);
 
+		// Decade
+		$newUnit = UnitOfMeasure::linearUnitFactory('dc', 315569520);
+		$newUnit->addAlias('decade');
+		$newUnit->addAlias('decades');
+		static::addUnit($newUnit);
+
+		// Century
+		$newUnit = UnitOfMeasure::linearUnitFactory('cnt', 3155695200);
+		$newUnit->addAlias('century');
+		$newUnit->addAlias('centuries');
+		static::addUnit($newUnit);
+
+		// Millennium
+		$newUnit = UnitOfMeasure::linearUnitFactory('mln', 31556952000);
```

Comment:
yes, I apologize for that, I was accounting for some misspellings on my app and forgot I committed those.  I'll handle the misspellings in other code though.  I built a Slack bot that interacts with an API of mine, and I thought it might be fun to have it do unit conversions and stuff too, so I can say "How many inches are in twenty-five kilometers?" and it shoots out the answer (with a few variations and misspellings allowed on the wording, like convert something to something, etc).  Anyhow, I will correct that.